pandas/core/resample.py

```diff
@@ -795,7 +795,7 @@ def std(self, ddof=1, *args, **kwargs):
         Parameters
         ----------
         ddof : integer, default 1
-        degrees of freedom
+            degrees of freedom
```

Comment:
Make sure the start of the description is always capitalized and ends with a period; otherwise you would be creating a new issue